frontera/contrib/scrapy/schedulers/frontier.py

```diff
@@ -35,38 +36,38 @@ def __init__(self, stats, prefix=STATS_PREFIX):
         self.prefix = prefix
 
     def add_seeds(self, count=1):
-        self._inc_value('seeds_count', count)
+        self.inc_value('seeds_count', count)
 
     def add_crawled_page(self, status_code, n_links):
-        self._inc_value('crawled_pages_count')
-        self._inc_value('crawled_pages_count/%s' % str(status_code))
-        self._inc_value('links_extracted_count', n_links)
+        self.inc_value('crawled_pages_count')
+        self.inc_value('crawled_pages_count/%s' % str(status_code))
+        self.inc_value('links_extracted_count', n_links)
 
     def add_redirected_requests(self, count=1):
-        self._inc_value('redirected_requests_count', count)
+        self.inc_value('redirected_requests_count', count)
 
     def add_returned_requests(self, count=1):
-        self._inc_value('returned_requests_count', count)
+        self.inc_value('returned_requests_count', count)
 
     def add_request_error(self, error_code):
-        self._inc_value('request_errors_count')
-        self._inc_value('request_errors_count/%s' % str(error_code))
+        self.inc_value('request_errors_count')
+        self.inc_value('request_errors_count/%s' % str(error_code))
 
     def set_iterations(self, iterations):
-        self._set_value('iterations', iterations)
+        self.set_value('iterations', iterations)
 
     def set_pending_requests(self, pending_requests):
-        self._set_value('pending_requests_count', pending_requests)
+        self.set_value('pending_requests_count', pending_requests)
 
-    def _get_stats_name(self, variable):
-        return '%s/%s' % (self.prefix, variable)
-
-    def _inc_value(self, variable, count=1):
+    def inc_value(self, variable, count=1):
```

Comment:
As I see, you changed StatsManager API, and made it replaceable by outside code. For what reason one would replace default one?